posthog/queries/breakdown_props.py

```diff
@@ -146,7 +159,10 @@ def get_breakdown_prop_values(
         filter.breakdown_group_type_index,
         filter.hogql_context,
         filter.breakdown_normalize_url,
-        direct_on_events=True if person_properties_mode == PersonPropertiesMode.DIRECT_ON_EVENTS else False,
+        direct_on_events=True
+        if person_properties_mode
+        in [PersonPropertiesMode.DIRECT_ON_EVENTS, PersonPropertiesMode.DIRECT_ON_EVENTS_WITH_POE_V2]
+        else False,
```

Comment:
I guess both of us missed this (maybe we both were just using replace tools) but `True if x else False` :DD

Reply:
dang it 😂